CONTRIBUTING.md

```diff
@@ -45,20 +45,30 @@ where `combo` is the name of the combination defined in [dazzle.yaml](dazzle.yam
 
 ### Build Specific Chunks
 
-Often, you would want to test only the chunks that you modify. You can do that by using the `-c` flag.
+Often, you would want to test only the chunks that you modify. You can do that with build-chunk.sh using the `-c` flag and specifying the name for your combination with `n`.
 
 ```console
-./dazzle-up.sh -c lang-c -c dep-cacert-update -n mychangecombo
+./build-chunk.sh -c lang-c -c dep-cacert-update -n mychangecombo
 ```
 
 Above command will build only chunks `lang-c` and `dep-cacert-update` and combine the created chunks (all variants, if any exists) as a combination with name `mychangecombo`.
 
+### Build Specific Combination
+
+Sometimes you only want to build one specification combination e.g. the postgresql or the go image. You can do that with 
```

Comment:
Sometimes you only want to build one specif combination e.g. the `postgresql` or the `go` image. You can do that with